src/librustc_resolve/lib.rs

```diff
@@ -1434,6 +1434,9 @@ pub struct Resolver<'a, 'b: 'a> {
     /// it's not used during normal resolution, only for better error reporting.
     struct_constructors: DefIdMap<(Def, ty::Visibility)>,
 
+    /// Map from tuple struct's DefId to VariantData's Def
+    tuple_structs: DefIdMap<Def>,
```

Comment:
With https://github.com/rust-lang/rust/pull/53751/files?w=1#r214416891 in mind, this field looks like an exact copy of `struct_constructors` field above.

Reply:
Thanks. It is removed.